ui/imports/utils/Utils.qml

```diff
@@ -938,4 +938,12 @@ QtObject {
     function isPunct(c) {
         return /(!|\@|#|\$|%|\^|&|\*|\(|\)|\+|\||-|=|\\|{|}|[|]|"|;|'|<|>|\?|,|\.|\/)/.test(c)
     }
+
+    function addTimestampToURL(url) {
```

Comment:
You have got the same function in two places: qml and nim.
Please, add a new function to global/utils.nim and use addTimestampToURL there. 
global/utils.nim object is exposed to qml. This way you will have this function in one place only.
Also, can you add a test/nim for this function?

Reply:
Fixed, I still trying to make work the adding of Unittest for the utils function as there was no infrastructure for the test. I need some more time to get the Makefile updated as I am getting link error while compile test file involving the Utils.nim file.

```
/usr/bin/ld: nimcache/debug/utils_test/@m..@s..@svendor@snim-status-go@sstatus_go.nim.c.o: in function `isAlias__OOZOOZvendorZnim45status45goZstatus95go_84':
/home/jacques/Documents/work/status-desktop/vendor/nim-status-go/status_go.nim:108: undefined reference to `IsAlias'
/usr/bin/ld: nimcache/debug/utils_test/@m..@s..@svendor@snim-status-go@sstatus_go.nim.c.o: in function 
...
```

In the meantime, the image URL received look like : 

```
https://Localhost:35651/contactImages?imageName=thumbnail&publicKey=0x04d76f84e8d041e733456e71eb811d486e16a51fd94fa8d5ffde9d39ed902317bc8ae2f632b562e716ce4b2fca1d7dc54420c412658167a7d0cecb2fb1de23407c
```

then we are just adding a `&timestamp=1231243242342` or `?timestamp=1231243242342` at the end.

Those query params are silent.

Reply:
nvm, thanks for your proposal offline to actually keep the time_utils.nim just call it from the global utils 👍🏾